test/fixtures/temp-cache-fixture/a.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function a() {
```

Comment:
These files should not be commited. They are only create temporary by the test suite. You probably canceled tests before they are remove. If you want to you could add them to `.gitignore`